R/MsExperiment.R

```diff
@@ -0,0 +1,63 @@
+#'@include PlainTextParam.R
+#'@title Methods to save and load contents of a MsExperiment object
+#'
+#' @description
+#'
+#' For `MsExperiment`, the exported data and related text files are:
```

Comment:
I would suggest to add a link to the `MsExperiment` package here. Something like *For MsExperiment objects, defined in the MsExperiment<link to github repo> package, the exported ...*. Same above for `MsBackendMzR` and then also for `XcmsExperiment`.